x-pack/plugins/observability_solution/entity_manager/server/routes/enablement/check.ts

```diff
@@ -76,9 +77,13 @@ export function checkEntityDiscoveryEnabledRoute<T extends RequestHandlerContext
               esClient,
               soClient,
               id: builtInDefinition.id,
+              includeState: true,
```

Comment:
I'm not sure we want the check endpoint to collect stats, isn't the goal here mostly to check the permissions and API key stuff?

Reply:
We fetch the state because we try to detect partial/corrupt installation and trigger a reinstall if components are missing